torch/_dynamo/variables/torch.py

```diff
@@ -852,6 +852,7 @@ def speculate_subgraph(f, sub_args, graph_checkpoint, checkpoint):
                         f"at {code.co_filename}:{code.co_firstlineno} because "
                         f"it closes over variables {closure_vars}. Please rewrite "
                         f"'{code.co_name}' to take {closure_vars} as additional args.",
+                        ref_case_id=26,
```

Comment:
These case ids need to be managed better. OK for now, but we need to have some kind of mapping, say from qualified names of tests, perhaps, to these numbers. Anyway I don't like the numbers showing up in export db as well.

Reply:
yeah we can talk about the naming system later.